classes/autoloader.php

```diff
@@ -41,7 +41,17 @@ public function __construct(array $namespaces = array(), array $namespaceAliases
 			$this->addNamespaceAlias($alias, $target);
 		}
 
-		foreach ($classAliases ?: array('atoum' => __NAMESPACE__ . '\test', __NAMESPACE__ => __NAMESPACE__ . '\test') as $alias => $target)
+		$defaultAliases = array(
+			'atoum' => __NAMESPACE__ . '\test',
+			__NAMESPACE__ => __NAMESPACE__ . '\test'
+		);
+
+		// Remove BC break for PHP_VERSION lower than 7.1.0 void is configured to be an alias of blank.
+		if (version_compare(PHP_VERSION, '7.1.0alpha2') < 0) {
```

Comment:
Completely agree, and I confirm this will introduce a BC Break. I agree with you the alias is not really useful here. And, I agree with you again the only patch valid is to rename the class void et don't provide any alias.

So do you need any help to valid or merge all PR ready for 2.x ?